static/js/reducers/index.js

```diff
@@ -141,9 +145,28 @@ export const profiles = (state: Profiles = INITIAL_PROFILES_STATE, action: Actio
       // caller must have dispatched START_PROFILE_EDIT successfully first
       return state;
     } else {
+      let errors = {};
+      let visibility = profile.edit.visibility;
+      if ( R.isEmpty(visibility) || R.contains(ALL_ERRORS_VISIBLE, visibility) ) {
```

Comment:
I think `R.isEmpty` is unnecessary here, since `visibility` is always set to an empty list initially and `edit` already exists